build.sbt

```diff
@@ -33,16 +33,16 @@ libraryDependencies ++= Seq(
   ("org.scorexfoundation" %% "scorex-core" % scorexVersion).exclude("ch.qos.logback", "logback-classic"),
 
   "org.typelevel" %% "cats-free" % "1.6.0",
-  "javax.xml.bind" % "jaxb-api" % "2.+",
-  "com.iheart" %% "ficus" % "1.4.+",
+  "javax.xml.bind" % "jaxb-api" % "2.4.0-b180830.0359",
+  "com.iheart" %% "ficus" % "1.4.7",
```

Comment:
Why this is needed?

Reply:
@aslesarenko It's not needed, I just "moved" here the exact versions that were in `lock.sbt`.